audio-worklet/design-pattern/lib/wasm-audio-helper.js

```diff
@@ -1,27 +1,20 @@
 /**
- * @fileOverview WA2 library. A set of utilities to support WASM operations
- * for the WebAudio. (WebAudio and WebAssembly => WAWA => WA2)
+ * Copyright (c) 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
```

Comment:
While valid, should we not use '//'-style comments instead of C-style?  Here and elsewhere.

Reply:
I think we should follow what DevRel team does. See the example here: https://github.com/GoogleChromeLabs/audioworklet-polyfill/blob/master/src/realm.js

FYI: Based on the suggestion from DevRel, I am changing the license to Apache 2.